.github/workflows/chain-interaction.yml

```diff
@@ -50,20 +50,22 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: desmos-labs/desmos
-          ref: refs/tags/v4.7.1
+          #ref: paul/set-stargate-querier-accepted-list
```

Comment:
I can find this branch on the desmos repo.

Reply:
Branch `paul/set-stargate-querier-accepted-list` was merged into master so it should be without ref now.